assets/android/src/SemanticColors.kt

```diff
@@ -113,6 +119,38 @@ Elevation: Level 0. */
     val borderInteractiveSecondary: Color,
     /** Subtle border colour for success state elements. */
     val borderSuccessSubtle: Color,
+    /** Background gradient stop for super and send buttons */
+    val gradientActionStop1: Color,
+    /** Background gradient stop for super and send buttons */
+    val gradientActionStop2: Color,
+    /** Background gradient stop for super and send buttons */
+    val gradientActionStop3: Color,
+    /** Background gradient stop for super and send buttons */
+    val gradientActionStop4: Color,
```

Comment:
@JordanConnor are you sure we can use this 4 colors for the super button? Current Super Button is 
<img width="218" alt="image" src="https://github.com/user-attachments/assets/4f2e80c2-ea8d-4151-b240-8be80ab1e161" />

and with those 4 colors, I get:

<img width="210" alt="image" src="https://github.com/user-attachments/assets/5dc77a54-2652-48ef-8f47-225e3e452633" />

CC @pixlwave

Reply:
I've adopted them already on the iOS side: https://github.com/element-hq/compound-ios-snapshots/commit/2fdf4f6267a2196f279d26e1bcd5faea23b47d17

The gradient now goes top to bottom now too based on the designs in the [Figma](https://www.figma.com/design/PpKepmHKGikp33Ql7iivbn/Compound-Styles?node-id=2071-1339&m=dev)